Lib/asyncio/unix_events.py

```diff
@@ -94,8 +93,9 @@ def add_signal_handler(self, sig, callback, *args):
         Raise ValueError if the signal number is invalid or uncatchable.
         Raise RuntimeError if there is a problem setting up the handler.
         """
-        if (coroutines.iscoroutine(callback) or
-                coroutines.iscoroutinefunction(callback)):
+        import inspect
+        if (inspect.iscoroutine(callback) or
```

Comment:
use `coroutines.iscoroutine` and `coroutines._iscoroutinefunction`

Reply:
Thanks!